### ISSUE TITLE
[Open In Sidebar] Userscripts won't work

### ISSUE THREAD

**Issue by justanotheranonymoususer** [chandler-stimson/color-picker#1]
Sorry for posting here, all links [here](https://webextension.org/listing/open-in-sidebar.html) lead to this repo and I didn't find any better place.

Open In Sidebar is very cool! Sadly userscripts don't work in the sidebar. Is it a Chrome limitation? I tried several userscript extensions, tampermonkey, violentmonkey...

Also posted here: https://github.com/Tampermonkey/tampermonkey/issues/1740#issuecomment-1656362742

**Comment by chandler-stimson** [comment]
The link has been updated to:
https://github.com/chandler-stimson/open-in-sidebar

The content displayed in the sidebar is embedded within an iframe element within the extension context. So other extensions cannot access this side panel. However, with the upcoming release, I have included an option page that allows you to write custom scripts. These scripts will be executed on each loaded page, effectively serving as a simple replacement for user scripts.
